pkg/monitortests/node/watchnodes/node.go

```diff
@@ -73,6 +85,15 @@ func startNodeMonitoring(ctx context.Context, m monitorapi.RecorderWriter, clien
 						WithAnnotation(monitorapi.AnnotationRoles, nodeRoles(node)).
 						HumanMessage("node is ready")).Build(now, now),
 			}
+
+		case wasUnknown && isUnknown:
+			return []monitorapi.Interval{
+				monitorapi.NewInterval(monitorapi.SourceNodeMonitor, monitorapi.Info).
+					Locator(monitorapi.NewLocator().NodeFromName(node.Name)).
+					Message(monitorapi.NewMessage().Reason("Unknown").
+						WithAnnotation(monitorapi.AnnotationRoles, nodeRoles(node)).
+						HumanMessage("node status is unknown")).Build(now, now),
+			}
```

Comment:
instead of going unknown here, when we transition to NotReady, can we try to determine if it is an expected notready or an unexpected not ready.

if we can't do it here, we should be able to do it in the analysis, but that should only be required there if we need additional resource/interval insights.